driver/ppm_fillers.c

```diff
@@ -7919,3 +7919,43 @@ int f_sys_prctl_x(struct event_filler_arguments *args)
 
 	return add_sentinel(args);
 }
+
+int f_sys_pidfd_getfd_x(struct event_filler_arguments *args){
+	unsigned long val;
+	int res;
+	int retval;
+
+	/*
+	* fd
+	*/
+	retval = (int64_t) syscall_get_return_value(current,args->regs);
+	res = val_to_ring(args, retval, 0, false, 0);
+	if(unlikely(res != PPM_SUCCESS))
+		return res;
```

Comment:
```suggestion
	CHECK_RES(res);
```
Plus the same comments of the other drivers